web-api/openapi/schemas/document.yml

```diff
@@ -32,6 +32,14 @@ DocumentTag:
   maxLength: 256
   example: "tag"
 
+DocumentSearchQuery:
+  description: A search query can be any non-empty, UTF-8-encoded string which doesn't contain a zero byte.
+  type: string
+  pattern: ".+"
+  minLength: 1
+  maxLength: 512
+  example: "what are the latest developments surrounding transformer models?"  
```

Comment:
:question: I'm not sure how well this kind of questions work with the embedding? Id not so well maybe we should use a different example.

Reply:
Was thinking about a plain "Lorem ipsum...", but updated the example to be more like a headline now